assets/src/data/stopsRed.ts

```diff
@@ -1,33 +1,161 @@
+import { LocationType } from "../models/stopData"
 import { Stop } from "../schedule"
 
 const stopsRed: Stop[] = [
-  { id: "place-alfcl", name: "Alewife", lat: 42.395428, lon: -71.142483 },
-  { id: "place-andrw", name: "Andrew", lat: 42.330154, lon: -71.057655 },
-  { id: "place-asmnl", name: "Ashmont", lat: 42.284652, lon: -71.064489 },
-  { id: "place-brdwy", name: "Broadway", lat: 42.342622, lon: -71.056967 },
-  { id: "place-brntn", name: "Braintree", lat: 42.207854, lon: -71.001138 },
-  { id: "place-chmnl", name: "Charles / MGH", lat: 42.361166, lon: -71.070628 },
-  { id: "place-cntsq", name: "Central", lat: 42.365486, lon: -71.103802 },
-  { id: "place-davis", name: "Davis", lat: 42.39674, lon: -71.121815 },
+  {
+    id: "place-alfcl",
+    name: "Alewife",
+    lat: 42.395428,
+    lon: -71.142483,
+    locationType: LocationType.Station,
+  },
+  {
+    id: "place-andrw",
+    name: "Andrew",
+    lat: 42.330154,
+    lon: -71.057655,
+    locationType: LocationType.Station,
+  },
+  {
+    id: "place-asmnl",
+    name: "Ashmont",
+    lat: 42.284652,
+    lon: -71.064489,
+    locationType: LocationType.Station,
+  },
+  {
+    id: "place-brdwy",
+    name: "Broadway",
+    lat: 42.342622,
+    lon: -71.056967,
+    locationType: LocationType.Station,
+  },
+  {
+    id: "place-brntn",
+    name: "Braintree",
+    lat: 42.207854,
+    lon: -71.001138,
+    locationType: LocationType.Station,
+  },
+  {
+    id: "place-chmnl",
+    name: "Charles / MGH",
+    lat: 42.361166,
+    lon: -71.070628,
+    locationType: LocationType.Station,
+  },
+  {
+    id: "place-cntsq",
+    name: "Central",
+    lat: 42.365486,
+    lon: -71.103802,
+    locationType: LocationType.Station,
+  },
+  {
+    id: "place-davis",
+    name: "Davis",
+    lat: 42.39674,
+    lon: -71.121815,
+    locationType: LocationType.Station,
+  },
```

Comment:
thought(non-blocking): I'm not sure this is a good idea, but while working on the vpc I started wondering if we could share our fishery factories with the frontend code.

I'm curious what @lemald thinks of that as well

what I was seeing was that the `place-` prefix and the `locationType` both seem like things that'd do well being specified as higher order functions around a base constructor.

and then also leave off a `id` if it was okay to just `toLower()` the `name` 🤣

Reply:
An interesting thought! FWIW this data is hardcoding what would be returned from the API. For example, `place-davis` is the id of the Davis square station in GTFS (https://www.mbta.com/stops/place-davis is the URL for davis stop information). These hardcoded stop + shape files in Skate will be removed as part of [this ticket ](https://app.asana.com/0/1152340551558956/1203629824340404). I'd be interested to hear more about the other cases you've come across in your VPC work though!